FAQ.md

```diff
@@ -976,4 +976,4 @@ The solution is to use the `--local-infile=1` option when you start the MySQL cl
 
 #### ERROR 9001 (HY000): PD server timeoutstart timestamp may fall behind safepoint
 
-This error occurs when TiDB accesses PD. A worker in the TiDB background continuously queries the safepoint in PD and this error occurs if it fails to query within 100s. Generally it is because PD failure or the network problem between TiDB and PD. For the details of common errors, see [Error Number and Fault Diagnosis](sql/error.md).
+This error occurs when TiDB fails to access PD. A worker in the TiDB background continuously queries the safepoint in PD and this error occurs if it fails to query within 100s. Generally it is because the PD failure or network failure between TiDB and PD. For the details of common errors, see [Error Number and Fault Diagnosis](sql/error.md).
```

Comment:
A worker in the TiDB background continuously queries the safepoint in PD -> **A worker continuously queries the safepoint from PD in the TiDB background**